docs/kubernetes/cross-cluster.md

```diff
@@ -0,0 +1,229 @@
+    ```console
+    $ kubectl exec web-server-deployment-cluster-b-6fc54d4776-gl5nl -- touch /usr/share/nginx/html/testfile
+    $ kubectl exec web-server-deployment-cluster-b-6fc54d4776-gl5nl -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+
+    $ kubectl exec web-server-deployment-cluster-b-6fc54d4776-jdx4f -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+
+    $ kubectl exec web-server-deployment-cluster-b-6fc54d4776-jn766 -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+    ```
+
+    Switch context to `ClusterA`. Check that the deployments in the `ClusterA`, pointing to the same filestore instance, can access the newly created file.
+    ```console
+    $ kubectl exec web-server-deployment-cluster-a-6fc54d4776-7lzkb -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+
+    $ kubectl exec web-server-deployment-cluster-a-6fc54d4776-nspm7 -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+
+    $ kubectl exec web-server-deployment-cluster-a-6fc54d4776-qdttl -- ls /usr/share/nginx/html
+    lost+found
+    testfile
+    ```
+
+5. Deletion of workloads and the volume resources.
+
+    Since the same filestore instance is shared across clusters, and the workload in one cluster is unaware of the filestore instance being shared by a different cluster, care must be taken by the user to ensure that no running workload is pointing filestore instance at the time of deletion.
+
+    Delete the pods in all the cluster
+    ```console
+    $ kubectl delete deployment web-server-deployment-cluster-a
+    deployment.apps "web-server-deployment-cluster-a" deleted
+    ```
+
+    Ensure the PersistentVolumeClaim (PVC) still exists in `ClusterA`
+    ```console
+    $ kubectl get pvc test-pvc-fs
+    NAME          STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+    test-pvc-fs   Bound    pvc-88959393-01c7-415f-aac4-a974c45f7ec7   1Ti        RWX            csi-filestore   27m
+    ```
+
+    Switch to `clusterB` and delete pods.
+    ```console
+    $ kubectl delete deployment web-server-deployment-cluster-b
+    deployment.apps "web-server-deployment-cluster-b" deleted
+    ```
+
+    Ensure PVC still exists in `ClusterB`
+    ```console
+    $ kubectl get pvc test-pvc-fs
+    NAME          STATUS   VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+    test-pvc-fs   Bound    pre-pv   1Ti        RWX            csi-filestore   7m57s
+    ```
+
+    At this point no running pod in any cluster is using the filestore instance. We can proceed with deletion of the PVC. In this example we choose to delete the PVC from the `clusterA` first. Since the PV has a Retain policy of `Retain`, deletion of PVC does not trigger the deletion of PersistentVolume or the filestore instance will . The PV enters a `Released` phase when the PVC is deleted. For more details into retain policy see [here](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming).
```

Comment:
garbled line around "the filestore instance will ."

Reply:
Fixed it